package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@silintl/ui-components",
-  "version": "1.0.0",
+  "version": "1.0.2",
```

Comment:
```suggestion
  "version": "1.1.0",
```

Reply:
Good call. It won't break the way they were being imported so I think minor version is appropriate as well.